_docs/community.md

```diff
@@ -3,16 +3,19 @@
 - [Community Contributions](#community-contributions)
   - [Agile Price Table](#agile-price-table)
   - [Export Rates Chart](#export-rates-chart)
+  - [Import And Export Rates Chart](#import-and-export-rates-chart)
 
 These are a few contributions by the community.
 
 ## Agile Price Table
 
-If you're wanting to display upcoming prices in a nice readable format, then might I suggest you utilise the plugin developed by @lozzd available at https://github.com/lozzd/octopus-energy-rates-card.
+If you're wanting to display upcoming prices in a nice readable format, then might I suggest you utilise the [Octopus Engergy Rates card](https://github.com/lozzd/octopus-energy-rates-card).
```

Comment:
Could you add the credit to @lozzd back in please?

Reply:
Done - removed by accident.